src/Racing.js

```diff
@@ -0,0 +1,37 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { MIN_RANDOM_SCORE } from './constants/index.js';
+
+class Racing {
+  #totalRound = 0;
+
+  #racers = [];
+
+  constructor({ totalRound, racers }) {
+    this.#totalRound = totalRound;
+    this.#racers = racers;
+  }
+
+  start() {
+    for (let i = 0; i < this.#totalRound; i += 1) {
+      this.#moveForward();
+      Console.print(this.#printResultByRound());
+    }
+    return this;
+  }
+
+  #moveForward() {
+    this.#racers.forEach((racer) => {
+      if (Random.pickNumberInRange(0, 9) >= MIN_RANDOM_SCORE) {
```

Comment:
```suggestion
      if (Random.pickNumberInRange(0, 9) >= MOVE_CONDITION) {
```
`MIN_RANDOM_SCORE`보다는 전진 조건을 나타내는 변수명이 조금 더 의도를 파악하기 좋을 것 같아요!

Reply:
고차 함수 활용을 잘하시네요!💯💯 

moveForward 로직은 Racer 클래스 안에 있어도 Racer 한 명에 관한 메서드라 잘 어울렸을 것 같습니다! 물론 여기서 addScore 해도 괜찮구요. 개인의 취향인 것 같네요😀